server/pbench/bin/pbench-sync-satellite

```diff
@@ -123,11 +147,11 @@ for host in $hosts ;do
     fi
 
     # move the unpacked files from tmp directory to archive
-    mv $unpack/$host/{.,}* .
+    mv $unpack/$host/* $unpack/$host/.prefix* .
     rc=$?
     if [[ $rc != 0 ]] ;then
-        echo "FAILED:    mv $unpack/$host/* ."
-        let failures=failures+1
+        echo "FAILED:    mv $unpack/$host/* $unpack/$host/.prefix* ."
+        exit 1
```

Comment:
I don't think this is the right action. This move failed, so maybe we leave it alone for now and try again in the next iteration, but why should we not continue? Let's talk about these points.